tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -144,97 +178,123 @@ def language_menu_item_description(self):
     def bluetooth_menu_item_description(self):
         return self.marionette.find_element(*self._bluetooth_text_locator).text
 
-    def a11y_open_accessibility_settings(self):
-        from gaiatest.apps.settings.regions.accessibility import Accessibility
-        self._a11y_click_menu_item(self._accessibility_menu_item_locator)
-        return Accessibility(self.marionette)
+    def open_wifi_settings(self):
```

Comment:
That looks :+1: ! We could reduce the amount of functions defined by defining them dynamically, but that would be overkill for now :)